docs/platform/dedicated-cpu/getting-started-with-dedicated-cpu/index.md

```diff
@@ -35,29 +37,35 @@ For more information on Dedicated CPU use cases, see our [Dedicated CPU Use Case
 
 1. Log in to the [Linode Cloud Manager](https://cloud.linode.com).
 
-2. Click on the **Create** dropdown menu at the top left of the page, and select "Linode".
+1. Click on the **Create** dropdown menu at the top left of the page, and select the **Linode** option.
 
-3. Select a [Distribution](/docs/quick-answers/linux/choosing-a-distribution/), [One-Click App](/docs/platform/one-click/how-to-use-one-click-apps-at-linode/), or [Image](/docs/platform/disk-images/linode-images/) to deploy from.
+1. Select a [Distribution](/docs/quick-answers/linux/choosing-a-distribution/), [One-Click App](/docs/platform/one-click/how-to-use-one-click-apps-at-linode/), or [Image](/docs/platform/disk-images/linode-images/) to deploy from.
 
     {{< note >}}
 Use a [StackScript](https://www.linode.com/stackscripts) to quickly deploy software platforms and system configuration options to your Linux distribution. You can read more about Stackscripts and how they work in our [Automating Deployments with Stackscripts Guide](/docs/platform/stackscripts/).
   {{< /note >}}
 
-1. Choose the region where you would like your Linode to reside. If you’re not sure which to select, see our [How to Choose a Data Center guide](/docs/platform/how-to-choose-a-data-center/). You can also generate [MTR reports](/docs/networking/diagnostics/diagnosing-network-issues-with-mtr/) for a deeper look at the route between you and a data center in each specific region.
+1. Choose the region where you would like your Linode to reside. If you’re not sure which to select, see our [How to Choose a Data Center guide](/docs/platform/how-to-choose-a-data-center/). You can also generate [MTR reports](/docs/networking/diagnostics/diagnosing-network-issues-with-mtr/) for a deeper look at the network route between you and each of our data centers.
 
-2. At the top of the **Linode Plan** section, click on *Dedicated CPU* and select the Dedicated CPU plan you would like to use.
+1. At the top of the **Linode Plan** section, click on the **Dedicated CPU** tab and select the Dedicated CPU plan you would like to use.
 
-3. Enter a label for your new Linode.
+1. Enter a label for your new Linode under the **Linode Label** field.
```

Comment:
Clarifying which field, as all the other steps make explicit reference to the field name